### DISCUSSION TITLE
Wizard Workflow doesn't change appearance with step advancement

### DISCUSSION THREAD

**Discussion by rseydler** [iTwin/iTwinUI-react#395] [Q&A]
I was wondering why the Wizard workflow doesn't change appearance like all the others do when you change step

direct link to the wizard workflow example here https://itwin.github.io/iTwinUI-react/?path=/story/core-wizard--workflow-diagram&args=currentStep:2

Thanks!

**Comment by bentleyvk** [comment]
Hi, it is intended to work like that. We are planning to have it as a separate component in the future because it doesn't really work like a true Wizard 🧙‍♂️. Here is the UX guidelines:
![image](https://user-images.githubusercontent.com/36186912/138215633-f88614e3-8c52-4c18-855d-553dfd35660c.png)
